lib/parseTriggers.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var FirebaseError = require('./error');
+var fork = require('child_process').fork;
+var path = require('path');
+var RSVP = require('rsvp');
+
+var TRIGGER_PARSER = path.resolve(__dirname, './triggerParser.js');
+
+module.exports = function(projectId, dbInstance, sourceDir) {
+  return new RSVP.Promise(function(resolve, reject) {
+    var env = {
```

Comment:
I think wait until later, there's a bug out (37681517), but it's lower priority at the moment.